lib/networking/rndis_protocol.h

```diff
@@ -0,0 +1,307 @@
+/**
+ * \file rndis_protocol.h
```

Comment:
these rndis protocol parser is very handy, maybe we should include them into the stack as well. Though I don't have enough knowledge at the moment to decide. We could always move files around later on.